docs/Advanced-Usage/Debugging-and-Profiling-on-FPGA/TracerV.rst

```diff
@@ -24,9 +24,9 @@ Building a Design with TracerV
 -------------------------------
 
 In all FireChip designs, TracerV is included by default. Other targets can
-enable it by attaching a TracerV Bridge to the RISC-V trace port of one-or-more
-cores. By default, only the cycle number, instruction address, and valid bit
-are collected.
+enable it by attaching a TracerV Bridge to the RISC-V trace port of each core
+they wish to trace (there should be one bridge per core).  By default, only the
+cycle number, instruction address, and valid bit are pulled off the FPGA.
```

Comment:
I actually preferred the term "collected" rather than "pulled off the FPGA"

Reply:
The problem is that it's ambiguous what collected means as the other signals are available in the bridge, but i can revert it.